src/main/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerChildListValidator.java

```diff
@@ -54,12 +55,15 @@ public boolean isValidLocations(CaseWorkerProfile caseWorkerProfile, ConstraintV
         if (isNotEmpty(caseWorkerProfile.getLocations())
             && caseWorkerProfile.getLocations().size() > 1) {
             //@TO do remove getLocationName with Id problem with excel sheet
-            isValidLocations = negate(caseWorkerProfile.getLocations().get(0).getLocationName()
-                .equalsIgnoreCase(caseWorkerProfile.getLocations().get(1).getLocationName()));
+            isValidLocations = negate(
+                    ObjectUtils.nullSafeEquals(
+                            String.valueOf(caseWorkerProfile.getLocations().get(0).getLocationName()).toLowerCase(),
```

Comment:
why we need to use String.valueOf ?

Reply:
Without it, it'll throw a null pointer exception because you can't invoke toLowerCase() on a null value